.changelog/v0.37.5/improvements/2846-speedup-json-encoding.md

```diff
@@ -0,0 +1,2 @@
+- `[libs/json]` Lower the memory overhead of JSON encoding by using JSON encoders internally
```

Comment:
@andynog changelog entry should be in `unreleased` folder, not `v0.37.5`

Reply:
Fixed here: https://github.com/cometbft/cometbft/pull/2887/commits/b94e5757b9daa30d1dd5a0c15904fafecfde2c23